packages/pipeline/src/data_sources/slippage/index.ts

```diff
@@ -0,0 +1,55 @@
+import { fetchAsync } from '@0x/utils';
+
+const EDPS_BASE_URL = 'https://ethereum-dex-prices-service.production.airswap.io';
+const PRICE_BASE_URL = 'https://min-api.cryptocompare.com/data/price?tsyms=USD';
```

Comment:
I feel like this logic could just go into `ohlcv_external/crypto_compare.ts`? and then you can import that code into here.